builders/selector/builder.toml

```diff
@@ -0,0 +1,81 @@
+description = "Ubuntu 22.04 Jammy Jellyfish full image with buildpacks for Apache HTTPD, Go, Java, Java Native Image, .NET, NGINX, Node.js, PHP, Procfile, Python, and Ruby"
+
+[[buildpacks]]
+  uri = "docker://ghcr.io/swissdatasciencecenter/vscodium-buildpack/vscodium:0.2"
+  version = "0.2.0"
+
+[[buildpacks]]
+  uri = "../../buildpacks/kernel-installer"
+  version = "0.0.1"
+
+[[buildpacks]]
+  uri = "../../buildpacks/frontend-selector"
+  version = "0.0.1"
+
+[[buildpacks]]
+  uri = "../../buildpacks/jupyterlab"
+  version = "0.0.1"
+
+[[buildpacks]]
+  uri = "../../buildpacks/frontends"
+  version = "0.0.1"
+
+[[buildpacks]]
+  uri = "docker://gcr.io/paketo-buildpacks/go:4.13.11"
+  version = "4.13.11"
```

Comment:
Why do we need this one? Just in case some user has a go project and we want to bake their dependencies in the image? Same question with node below.

Reply:
Yes if someone has a go project they can use this buildpack